nasa/openmct
# New folder action

Adds a new action to allow users to create folders by context clicking other folders

## Testing Instructions
1. Right click on a folder from the tree
2. Confirm there is an option for a new folder
3. Click on the new folder option
4. Enter a folder name in the following dialog
5. Confirm a new folder was created, with the name you entered. 

## Author Checklist

1. Changes address original issue? Yes
2. Unit tests included and/or updated with changes? Yes
3. Command line build passes? Yes
4. Changes have been smoke-tested? Yes
5. Testing instructions included? Yes
